web/template_overrides/webclient/webclient.html

```diff
@@ -0,0 +1,68 @@
+{% extends "webclient/base.html" %}
+
+<!-- Django-template blocks available for overloading:
+    - connecting     - custom connect messages
+    - jquery_import  - for changing to a local jquery version
+    - guilib_import  - for using your own gui lib
+-->
+{% load staticfiles %}
+
+{% block jquery_import %}
+  <script src="{% static 'webclient/js/jquery-2.1.1.min.js' %}" type="text/javascript" charset="utf-8"></script>
```

Comment:
jQuery isn't loaded directly from the js folder - it's loaded via the code.jquery.com CDN, so this breaks the webclient

Reply:
Fixed!